static/src/javascripts/projects/facia/modules/ui/au-region-selector.ts

```diff
@@ -41,6 +41,7 @@ export const init = (): void => {
 					// fastly will ignore the cookie & use the users locations
 					const value =
 						attrValue === 'AU-hide-thrasher' ? 'Other' : attrValue;
+					removeCookie({ name: 'GU_territory' });
```

Comment:
Nitpicky but should we remove the old cookie after we've set the new one? It's an edge case but if the code to set the new one were to fail we'd want to keep that legacy cookie

Reply:
Hm, good point, although I'm not sure in what scenario we'd only be able to remove cookies and not add them! Our current cookie library doesn't really give us a great deal of control in this situation either, theres no option to filter by domain when clearing cookies so if we were to remove the cookie after setting it we'd need to not use @guardian/libs removeCookie function.